signer/jws_test.go

```diff
@@ -0,0 +1,148 @@
+package signer
+
+import (
+	"crypto/x509"
+	"errors"
+	"github.com/notaryproject/notation-core-go/internal/testhelper"
+	"testing"
+)
+
+func TestNewJWSEnvelopeFromBytes(t *testing.T) {
+	t.Run("newJWSEnvelopeFromBytes", func(t *testing.T) {
+		if _, err := newJWSEnvelopeFromBytes([]byte(TestValidSig)); err != nil {
+			t.Errorf("Error found")
```

Comment:
Print err? Also, good idea to print the test case name.

Reply:
yep updated.